src/AdoNetCore.AseClient/Internal/Handler/MessageTokenHandler.cs

```diff
@@ -19,17 +21,42 @@ public void Handle(IToken token)
             switch (token)
             {
                 case EedToken t:
-                    var isSevere = t.Severity > 10;
-                    var msgType = isSevere
-                        ? "ERROR"
-                        : "INFO ";
 
-                    var formatted = $"{msgType} [{t.Severity}] [L:{t.LineNumber}]: {t.Message}";
+                    var isBackupServerMessage = BackupServerErrorInfo.TryParse(t.Message, out BackupServerErrorInfo backupServerErrorInfo);
```

Comment:
I don't think these checks are necessary since they relates to parsing the messages.

Just checking the `Severity` field is fine